app/controllers/casa_cases_controller.rb

```diff
@@ -1,6 +1,6 @@
 class CasaCasesController < ApplicationController
   before_action :set_casa_case, only: %i[show edit update destroy deactivate reactivate]
-  before_action :set_contact_types, only: %i[new edit update create]
+  before_action :set_contact_types, only: %i[new edit update create deactivate]
```

Comment:
The `deactivate` method in this controller renders the `edit` template in the failure case. The `edit` template renders the `_form` partial and with this PR update, we are showing contact types for admins. 
One of the admin tests started [failing](https://github.com/rubyforgood/casa/runs/1916083905?check_suite_focus=true), and realized we need this change!